test/unit/language-extensions/multi.spec.ts

```diff
@@ -24,6 +24,19 @@ test("multi example use case", () => {
         .expectToEqual({ a: "foo", b: 5 });
 });
 
+test("multi #995", () => {
+    util.testModule`
+        function multiReturn(): LuaMultiReturn<[number, number, number]> {
+            return $multi(1, 2, 3);
```

Comment:
Could you add another test where this function returns nil? (like the string.find in the issue)